lib/glific/clients/stir.ex

```diff
@@ -145,6 +145,21 @@ defmodule Glific.Clients.Stir do
     %{message: praority_message, exculde: priority_map[exculde]}
   end
 
+  def webhook("fetch_other_priorities", fields) do
+    priority_map = Enum.into(@priorities_list, %{})
+    first_priority = fields["first_priority"] |> String.downcase()
+    second_priority = fields["second_priority"] |> String.downcase()
+
+    remaining_priorities =
+      priority_map
+      |> Map.delete(first_priority)
+      |> Map.delete(second_priority)
+
+    %{
+      remaining_priorities: remaining_priorities
```

Comment:
Let's simply results as much as we can, it helps in the flows
```suggestion
      remaining_priority_first: PNAME,
       remaining_priority_second: PNAME
```